architecture/topics/haproxy_template_router.adoc

```diff
@@ -27,4 +27,106 @@ The following diagram illustrates how data flows from the master through the
 plug-in and finally into an HAProxy configuration:
 
 .HAProxy Router Data Flow
-image::router_model.png[HAProxy Router Data Flow]
\ No newline at end of file
+image::router_model.png[HAProxy Router Data Flow]
+
+[[haproxy-metrics]]
+=== HAProxy Template Router Metrics
+
+The HAProxy router, by default, exposes or publishes metrics in
```

Comment:
"by default"

Not sure if we want to say that... why not drop it.

Reply:
Changed